rebar.lock

```diff
@@ -1 +1,16 @@
-[].
+{"1.1.0",
+[{<<"ebus">>,
+  {git,"https://github.com/helium/ebus",
+       {ref,"f1b415a11fb5a4d6cd5f18f82b141785b4bc49cc"}},
+  1},
```

Comment:
I take it that gateway_config can depend on a different version of ebus than gatt and both versions will coexist with each other.  Or does gateway_config's version of ebus supersede gatt's?